pages/products.html

```diff
@@ -0,0 +1,453 @@
+                                    <span class="btn-card-icon"><i class="fas fa-shopping-cart"></i>
+                           </span>Add to Cart</button>  
+                              </div>
+                            </div>
+                            <span class="tag">New</span>
+                          </div>              
+                    </li>
+                    <li class="section-main">
+                        <div class="cards card-vertical">
+                            <div class="img-container">
+                              <img
+                                src="../components/images/bags3.3.jpeg"
+                                alt="product image"
+                                class="grid-col-img cards-img"
+                              />
+                            </div>
+              
+                            <div class="card-content padding-s">
+                              <h3 class="card-grid">
+                                Stylish Bags <i class="fas fa-heart cards-icon"></i>
+                              </h3>
+              
+                              <span class="cards-desc">Girls White & Pink Bag</span>
+              
+                              <div class="gap-s">
+                                <span class="txt-bold"> Rs.3899</span>
+                                <span class="txt-crossed-off">Rs.2999</span>
+                                <span class="txt-high-light">10%</span>
+                              </div>
+
+                              <div class="card-footer">
+                                <button class="btn btn-text-icon-primary grid-cards-icons">
+                                    <span class="btn-card-icon"><i class="fas fa-shopping-cart"></i>
+                           </span>Add to Cart</button>  
+                              </div>
+                            </div>
+                            <span class="tag">New</span>
+                          </div>              
+                    </li>
+                    <li class="section-main">
+                        <div class="cards card-vertical">
+                            <div class="img-container">
+                              <img
+                                src="../components/images/bags3.3.jpeg"
+                                alt="product image"
+                                class="grid-col-img cards-img"
+                              />
+                            </div>
+              
+                            <div class="card-content padding-s">
+                              <h3 class="card-grid">
+                                Stylish Bags <i class="fas fa-heart cards-icon"></i>
+                              </h3>
+              
+                              <span class="cards-desc">Girls White & Pink Bag</span>
+              
+                              <div class="gap-s">
```

Comment:
Just a suggestion: Naming could be improved here. It could also be like `sm-gap` for small gap or `md-gap` for little more gap. I believe it would be more clear that way.